### PR TITLE
correct PHP code

### PR DESCRIPTION
**Pull request by MedUnes**

### CODE REVIEW

File: form/unit_testing.rst [hunk 1]
```diff
@@ -227,14 +227,15 @@ a good opportunity to use them::
 
     class TestedTypeTest extends TypeTestCase
     {
-        /**
-         * @dataProvider getValidTestData
-         */
+
```
Comment (apfelbox) [inline form/unit_testing.rst]: This change looks wrong, as the data provider for the method `testForm` is the method `getValidTestData`.

The PHP doc

```php
/**
 * @dataProvider getValidTestData
 */
```
doesn't mean

> **this** method is a data provider

but instead means

> the data provider **for this method** is this other method

Comment (apfelbox) [inline form/unit_testing.rst]: See https://phpunit.de/manual/6.2/en/writing-tests-for-phpunit.html#writing-tests-for-phpunit.data-providers

Comment (xabbuh) [inline form/unit_testing.rst]: Indeed, this part needs to be kept as is. But the other change is great. 👍

Comment (MedUnes) [inline form/unit_testing.rst]: thanks @apfelbox , it is clear that you are right
thanks @xabbuh, I'm happy to correct a Symfony's documentation for the second time :+1:

**Review by xabbuh — APPROVED** [review]

**Comment by xabbuh** [comment]
Thank you @MedUnes.

**Pull request closed without merging**